class-tgm-plugin-activation.php

```diff
@@ -1337,6 +1362,22 @@ protected function display_settings_errors() {
 		}
 
 		/**
+		 * Display admin notice if plugins have been force activated.
+		 *
+		 * @since 2.6.0
+		 */
+		public function display_forced_activation_notice() {
+			if ( $force_activated_plugins = get_transient( 'tgmpa_force_activated_plugins' ) ) {
+				?>
+				<div class="notice <?php echo esc_attr( $this->get_admin_notice_class() ); ?> is-dismissible">
```

Comment:
I've got no objection to this. By going out of PHP, you then benefit from syntax-highlighted markup, and it's the first step to moving markup out of the PHP logic and into view files.

What I'd like to see fixed is changing `esc_attr()` to `sanitize_html_class()` here.